source/Expression/ExpressionSourceCode.cpp

```diff
@@ -236,8 +239,16 @@ bool ExpressionSourceCode::SaveExpressionTextToTempFile(
     break;
   }
 
+#if defined(_WIN32)
```

Comment:
Yeah, this is ugly. Let me know about this function: http://llvm.org/docs/doxygen/html/namespacellvm_1_1sys_1_1fs.html#ae3cdedf952d5eeb089d175c61d9f6fee

I think it does what we want